src/lib/plugin_apis/lvm.api

```diff
@@ -2044,4 +2045,34 @@ gboolean bd_lvm_devices_delete (const gchar *device, const gchar *devices_file,
  */
 gchar* bd_lvm_config_get (const gchar *section, const gchar *setting, const gchar *type, gboolean values_only, gboolean global_config, const BDExtraArg **extra, GError **error);
 
+/**
+ * bd_lvm_vgcfgbackup:
+ * @vg_name: name of the VG to backup
+ * @backup_file: (nullable): file to save the backup to or %NULL for using the default backup file
+ *                           in /etc/lvm/backup
+ * @extra: (nullable) (array zero-terminated=1): extra options for the vgcfgbackup command
+ *                                               (just passed to LVM as is)
+ * @error: (out) (optional): place to store error (if any)
+ *
+ * Returns: Whether the backup was successfully created or not.
+ *
+ * Tech category: %BD_LVM_TECH_VG_BACKUP_RESTORE no mode (it is ignored)
+ */
+gboolean bd_lvm_vgcfgbackup (const gchar *vg_name, const gchar *backup_file, const BDExtraArg **extra, GError **error);
+
+/**
+ * bd_lvm_vgcfgrestore:
+ * @vg_name: name of the VG to restore
+ * @backup_file: (nullable): file to restore VG configuration from to or %NULL for using the
+ *                           latest backup in /etc/lvm/backup
+ * @extra: (nullable) (array zero-terminated=1): extra options for the vgcfgrestore command
+ *                                               (just passed to LVM as is)
+ * @error: (out) (optional): place to store error (if any)
+ *
+ * Returns: Whether the backup was successfully created or not.
```

Comment:
Perhaps add a sentence from the `vgcfgrestore(8)` manpage?

 _Restore volume group configuration_

_vgcfgrestore  restores  the metadata of a VG from a text back up file produced by vgcfgbackup._